youtube_dl/extractor/vier.py

```diff
@@ -67,14 +73,28 @@ def _real_extract(self, url):
         self._sort_formats(formats)
 
         title = self._og_search_title(webpage, default=display_id)
-        description = self._og_search_description(webpage, default=None)
```

Comment:
For some videos, the description extracted by `self._og_search_description(..)` is something like "Watch the complete Xth episode of Y here" (but in Dutch) instead of the actual description displayed below the video. I'll add a test for this.